.prettierrc.yaml

```diff
@@ -0,0 +1,6 @@
+singleQuote: false
+semi: false
+arrowParens: avoid
+trailingComma: es5
```

Comment:
Please set to `all`.

Reply:
trailingComma: es5 not includes function params